src/SFA.DAS.Commitments.Infrastructure/app.config

```diff
@@ -58,6 +58,10 @@
         <assemblyIdentity name="Microsoft.Extensions.Configuration.Abstractions" publicKeyToken="adb9793829ddae60" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-2.2.0.0" newVersion="2.2.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.Azure.Services.AppAuthentication" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-1.6.2.0" newVersion="1.6.2.0" />
+      </dependentAssembly>
```

Comment:
I had trouble with version 1.6.2 as it requires `Microsoft.IdentityModel.Clients.ActiveDirectory` >= 5.2.9 which conflicts with other packages

Reply:
Made all references the 1.0.1 package which is known to work elsewhere - and set the binding redirect to that version only